Library/Homebrew/cli/args.rb

```diff
@@ -35,11 +35,13 @@ def freeze_remaining_args!(remaining_args)
       def freeze_named_args!(named_args, cask_options:)
         self[:named] = NamedArgs.new(
           *named_args.freeze,
-          override_spec: spec(nil),
-          force_bottle:  self[:force_bottle?],
-          flags:         flags_only,
-          cask_options:  cask_options,
-          parent:        self,
+          parent:       self,
+          cask_options: cask_options,
+          **{
```

Comment:
Could this be pulled out into variable (and all the parenthesised bits below)? Will make it a bit easier to follow than embedding into params.